tools/rotate-certs.sh

```diff
@@ -22,6 +23,10 @@ while true; do
         shift;
         cdi_namespace=$1
         ;;
+    --webui-namespace)
```

Comment:
I think you have also to add `,webui-namespace:` on line 9

Reply:
Added, thank you. I am currently looking for an environment to run this script altogether, not just by snippets.